<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Export_Video_Widget</class>
 <widget class="QWidget" name="Export_Video_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>243</width>
    <height>130</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSpinBox" name="start_frame_spinbox"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Start Frame</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QSpinBox" name="end_frame_spinbox">
       <property name="minimum">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>End Frame</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="output_filename">
     <property name="text">
      <string>output.mp4</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="export_video_button">
     <property name="text">
      <string>Export!</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
